Automattic/Gravatar-SDK-Android
# Remove docs from trunk

Part of #371

### Description

This part removes the generated API documentation from the `trunk` branch. 

To be more precise, I removed:
- `docs/dokka/` directory
- docs/index.html` file


Those docs are already available in a separate branch `docs` and that is where the gh pages are configured to build from. 

Next, I will work on the automation, to build those docs automatically on each tag push.

### Testing Steps

There is nothing to test really, you can check the `docs` folder and see if the gh pages are working.
